### DISCUSSION TITLE
How to determine swig 4.0.1 linux package is compiled with -D_GLIBCXX_USE_CXX11_ABI=1 ?

### DISCUSSION THREAD

**Discussion by bored22** [swig/swig#2147] [Q&A] [answered]
Hi, This is my first post and I am fairly new to swig software.
I downloaded swig source from site-https://sourceforge.net/projects/swig/files/swig/swig-4.0.1/ and compiled with below commands:
./configure
make
sudo make install

now how do I check if this version was built with -D_GLIBCXX_USE_CXX11_ABI=1 ?
Thanks

**Comment by wsfulton** [comment, marked as answer]
The default compiler options that SWIG uses, do not add `-D_GLIBCXX_USE_CXX11_ABI=1` when compiling the SWIG executable after running `make`. If you want to know what ABI you compiled with, please refer to your compiler's documentation. I'm afraid that the SWIG project is not the correct place to advise.
